README.md

```diff
@@ -49,7 +103,7 @@ See [the tests](https://github.com/glennsl/bs-jest/tree/master/__tests__) for mo
 ## Installation
 
 ```sh
-npm install --save-dev @glennsl/bs-jest
+yarn install --save-dev @glennsl/bs-jest
```

Comment:
I suppose this should use `npm` too. Or show both.

```suggestion
npm install --save-dev @glennsl/bs-jest
```

Reply:
I updated my code to show both npm and yarn syntax (see [commit 0aaf292](https://github.com/glennsl/bs-jest/pull/96/commits/0aaf292b64117f5f77c6a448a3c80b64d7422413).  I suggest releasing this as @glennsl/rescript-jest rather than @glennsl/bs-jest because, from a semantic version perspective, this is a major breaking change but it might not be sufficiently complete from perspective of Jest API bindings to warrant updating to a 1.x.x version number.  I have never actually released a project to npm so I don't know what specifically would need to be done to make that happen.  I'll forward your lead.  Let me know your thoughts.